qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -246,8 +250,47 @@ def selection(self, html=False):
             raise browsertab.UnsupportedOperationError
         return self._widget.selectedText()
 
+    def _follow_selected_cb(self, js_elem, tab=False):
+        """Callback for javascript which clicks the selected element.
+
+        Args:
+            js_elems: The elements serialized from javascript.
+            tab: Open in a new tab or not.
+        """
+        if js_elem is None:
+            return
+        assert isinstance(js_elem, dict), js_elem
+        elem = webengineelem.WebEngineElement(js_elem, tab=self._tab)
+        if tab:
+            click_type = usertypes.ClickTarget.tab
+        else:
+            click_type = usertypes.ClickTarget.normal
+
+        # Only click if we see a link
+        if elem.is_link():
+            log.webview.debug("Found link in selection, clicking. Tab: " +
+                              str(click_type) + "Link: " + str(elem))
+            elem.click(click_type)
+
     def follow_selected(self, *, tab=False):
-        log.stub()
```

Comment:
You can remove `pylint: disable=unused-argument` at the beginning of the file now.